app/src/main/java/it/niedermann/owncloud/notes/main/navigation/NavigationViewHolder.java

```diff
@@ -59,5 +60,9 @@ public void bind(@NonNull NavigationItem item, @ColorInt int mainColor, String s
         icon.setColorFilter(isSelected ? textColor : 0);
 
         view.setSelected(isSelected);
+
+        ViewGroup.MarginLayoutParams params = (ViewGroup.MarginLayoutParams) view.getLayoutParams();
+        params.leftMargin = item.icon == NavigationAdapter.ICON_SUB_FOLDER ? 70 : 0;
```

Comment:
Please move this magic number (`70`) to `dimens.xml` and fetch it via `getResources().getDimensionPixelSize(R.dimen....)`

Reply:
yes I ve done it